addon/components/layers/group-layer.js

```diff
@@ -1,24 +1,49 @@
 /**
   @module ember-flexberry-gis
- */
+*/
 
-import BaseLayer from 'ember-flexberry-gis/components/base-layer';
+import BaseLayer from '../base-layer';
 import layout from '../../templates/components/layers/group-layer';
 
 /**
   GroupLayerComponent for leaflet map.
   @class GroupLayerComponent
   @extend BaseLayerComponent
- */
```

Comment:
removed spaces in documentation comments, you should create task for support team.